src/workers/jobs.js

```diff
@@ -1118,3 +1118,39 @@ export async function clearOldJobs(delay) {
     .where("updated_at", "<", delay)
     .delete();
 }
+
+export async function buyPhoneNumbers(job) {
+  try {
+    if (!job.organization_id) {
+      throw Error("organization_id is required");
+    }
+    const payload = JSON.parse(job.payload);
+    const { areaCode, limit, messagingServiceSid } = payload;
+    if (!areaCode || !limit) {
+      throw new Error("areaCode and limit are required");
+    }
+    const organization = await Organization.get(job.organization_id);
```

Comment:
please use `loaders` here, so cache can be used.

Reply:
`cacheableData`, not `loaders`, since we are outside of the request scope right?